docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -69,193 +53,71 @@ Use the toolbar buttons in the upper-left of the Alerts table to customize the c
-
-*Threat Intel*
-
-The *Threat Intel* tab shows the number of individual threats matching the alert. You can expand and collapse threat details by clicking the arrow button at the end of the threat label. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available.
-
-Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
-
-* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
-* *Enriched with Threat Intelligence*: This section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected time frame, the section displays a message that none are available.
-
-NOTE: The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
-
-*Table*
-
-The *Table* tab shows the alert details in table format. Alert details are organized into field value pairs.
-
-*JSON*
-
-The *JSON* tab shows the alert details in JSON format.
-
 [float]
 [[alert-actions]]
 === Take actions on an alert
-From the Alerts table or the Alert details flyout, you can do the following:
+From the Alerts table or the alert details flyout, you can:
 
-* <<signals-to-cases,Add alerts to a case>>
-* <<detection-alert-status,Change an alert's status>>
-* <<endpoint-rule-exceptions,Add an endpoint exception to an alert>>
-* <<add-exception-from-alerts,Add a rule exception to an alert>>
+* <<signals-to-cases>>
+* <<detection-alert-status>>
+* <<add-exception-from-alerts>>
+* <<endpoint-rule-exceptions,Add an endpoint exception from an alert>>
```

Comment:
@benironside Yes, exceptions are added either to rules or to the entire endpoint, but nothing is really done _to_ the alert itself. The alert is just one context _from_ which the user can create an exception, so that's why this heading calls out that particular context (as opposed to other contexts, like creating it from the rule details page, etc.).